{"uuid": "872896bc-f0a6-48af-9958-f5afa9478fa0", "children": ["0c358d08-aac5-4f0c-84a0-22159da092fd", "574fcfac-3470-4936-8667-8cc0779e889c"], "befores": [{"name": "browser", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\tests\\ui\\conftest.py\", line 13, in browser\n    playwright = sync_playwright().start()\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "steps": [{"name": "Запуск Playwright и инициализация браузера Chromium", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.\n", "trace": "  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\tests\\ui\\conftest.py\", line 13, in browser\n    playwright = sync_playwright().start()\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 84, in start\n    return self.__enter__()\n           ~~~~~~~~~~~~~~^^\n  File \"C:\\Users\\X\\PycharmProjects\\PythonProject\\clickup_tests\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py\", line 47, in __enter__\n                raise Error(\n    ...<2 lines>...\n                )\n"}, "start": 1747737819542, "stop": 1747737819542}], "start": 1747737819542, "stop": 1747737819543}], "start": 1747737819542, "stop": 1747737819558}